comment/tests/test_views/test_blocker.py

```diff
@@ -0,0 +1,96 @@
+from django.urls import reverse
+
+from comment.tests.base import BaseCommentMixinTest
+from comment.views import BaseToggleBlockingView
+from comment.messages import BlockedUserMSG
+from comment.models import BlockedUser
+
+
+class BaseToggleBlockingViewTest(BaseCommentMixinTest):
+    def setUp(self):
+        super().setUp()
+        self.view = BaseToggleBlockingView()
+        self.toggle_blocking_url = self.get_url(reverse('comment:toggle-blocking'))
```

Comment:
moving this to `setUpTestData` could/should/would potentially speed up the tests.